resources/event-sources/values.yaml

```diff
@@ -4,9 +4,9 @@ global:
   controllerManager:
     name: controller-manager
     image: event-sources-controller-manager
-    dir: develop/
-    version: #TODO
+    dir: ''
```

Comment:
I think we need to be consistent with the other images which have `develop/` in their path.

Reply:
The Prow job sends an empty dir and builds images like that. If I add that dir here, the image name becomes invalid. I think this changed with the shared Makefile.

Reply:
True